requirements.txt

```diff
@@ -3,6 +3,8 @@
 #
 opencv-python
 numpy
+pandas
+scipy
```

Comment:
What is the purpose of these imports in this PR?  I don't really want to expand the dependencies of the package without much discussion.